tests/backoff/Makefile

```
@@ -0,0 +1,18 @@
+all: ${PROG}
+
+${PROG}: ${SRCS}
+	${CC} ${CFLAGS} -o ${PROG} ${SRCS} ${LDFLAGS}
+
+test: ${PROG}
+	@echo "Run as root: sudo ./${PROG} [-b path-to-dhcpcd]"
+
+clean:
+	rm -f ${PROG}
+
+distclean: clean
```

checkmake (0.3.2)

[warning] Required target "all" must be declared PHONY.

(minphony)